sw/make-deb

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# --------------------------------------------------------------------------
+# Create deb-package after compile
```

Comment:
An update to README.md would be appreciated!

Reply:
yes, is in the pipeline :-)